packages/cysync-core/src/pages/MainApp/Components/MainAppLayout.tsx

```diff
@@ -1,28 +1,36 @@
-import { Container, Flex } from '@cypherock/cysync-ui';
+import { Flex } from '@cypherock/cysync-ui';
 import React, { FC, ReactNode } from 'react';
 
 import { AppUpdateBar, Topbar } from '~/components';
 
 import { SideBar } from './SideBar';
+import styled from 'styled-components';
 
 interface MainAppLayoutProps {
   title: string;
   children?: ReactNode;
 }
 
-export const MainAppLayout: FC<MainAppLayoutProps> = ({ title, children }) => (
-  <Container height="screen" display="flex" $bgColor="contentGradient">
+const MainAppLayoutStyle = styled.div<MainAppLayoutProps>`
+  display: flex;
+  width: 100%;
+  min-height: 100vh;
+  background: ${({ theme }) => theme.palette.background.content};
+`;
```

Comment:
Do not use styled components in cysync-core, You can use `Flex` component for this